app/controllers/transactions/transaction-detail.controller.js

```diff
@@ -19,6 +20,10 @@ module.exports = (req, res) => {
         delete req.session.backLink
       }
       data.service = req.service
+
+      const refundUrl = router.generateRoute(formatAccountPathsFor(router.paths.account.transactions.refund, req.account.external_id), { chargeId })
+
+      data.refundUrl = refundUrl
```

Comment:
I think @sfount mentioned before that this could go into the template rather than passing it in through the model. But seems fine if there's a particular reason this is easier.

Reply:
We couldn't get the nunjucks `replace` method to work.
So ended up adding it to the controller.
Let me know, if you know think of a better way.

Reply:
Unless I'm missing something this `refundUrl` isn't actually used by the template? is there any chance it does now work?

Reply:
Good spot - that is not used anymore.
Removing that line.